### PR TITLE
First implementation

### PR DESCRIPTION
**Pull request by banyan**
Specs are here: https://developer.sbpayment.jp/doc/tbx/API_BasicSpec.zip (it's only japanese though :bow:)

```
DEV banyan ☁  be ruby example/sample-request.rb
<?xml version='1.0' encoding='Shift_JIS' ?>
  <sps-api-response id="ST01-00101-101">
    <res_result>OK</res_result>
    <res_sps_transaction_id>B30132002ST010010110100121170876</res_sps_transaction_id>
    <res_tracking_id>00000013340929</res_tracking_id>
    <res_pay_method_info>
      <cc_company_code>99665</cc_company_code>

      <recognized_no>123456</recognized_no>

    </res_pay_method_info>
    <res_sps_info>
      <res_sps_cust_no/>
      <res_sps_payment_no/>
    </res_sps_info>
    <res_process_date>20150721121150</res_process_date>
    <res_err_code/>
    <res_date>20150721121150</res_date>
  </sps-api-response>
```

updated.

```
DEV banyan ☁  be ruby example/sample-request.rb
{"id"=>"ST01-00101-101",
 "res_result"=>["OK"],
 "res_sps_transaction_id"=>["B30132002ST010010110100121192300"],
 "res_tracking_id"=>["00000013341724"],
 "res_pay_method_info"=>
  [{"cc_company_code"=>["99665"], "recognized_no"=>["123456"]}],
 "res_sps_info"=>[{"res_sps_cust_no"=>[{}], "res_sps_payment_no"=>[{}]}],
 "res_process_date"=>["20150721172327"],
 "res_err_code"=>[{}],
 "res_date"=>["20150721172327"]}
```

### CODE REVIEW

**Comment by banyan** [comment]
It's still initial implementation. however, will enhance more in next PR. Could you review and merge? @quipper/developers

File: example/sandbox.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  params.each do |key, value|
+    if value.is_a? Hash
+      xml.tag! key do
+        value.each do |k, v|
+          xml.tag! k, v
+        end
+      end
+    else
+      if key == :item_name
+        xml.tag! key, Base64.encode64(value).strip
```
Comment (miyucy) [inline example/sandbox.rb]: `Base64.strict_encode64`

Comment (banyan) [inline example/sandbox.rb]: Thanks!

**Comment by banyan** [comment]
Updated. @miyucy

**Pull request merged**